### ISSUE TITLE
v1.2.1 compile error 

### ISSUE THREAD

**Issue by digoal** [kibae/onnxruntime-server#35]
```
/tmp/pg_onnx/onnxruntime-server/onnxruntime-server/src/onnx/value_info.cpp: In member function ‘nlohmann::json_abi_v3_11_2::basic_json<>::array_t onnxruntime_server::onnx::value_info::get_tensor_data(Ort::Value&) const’:
/tmp/pg_onnx/onnxruntime-server/onnxruntime-server/src/onnx/value_info.cpp:128:51: error: ‘const struct Ort::Float16_t’ has no member named ‘value’; did you mean ‘val’?
  128 |    values.emplace_back(float16_to_float32(data[i].value));
      |                                                   ^~~~~
      |                                                   val
/tmp/pg_onnx/onnxruntime-server/onnxruntime-server/src/onnx/value_info.cpp:135:52: error: ‘const struct Ort::BFloat16_t’ has no member named ‘value’; did you mean ‘val’?
  135 |    values.emplace_back(bfloat16_to_float32(data[i].value));
      |                                                    ^~~~~
      |                                                    val
gmake[2]: *** [src/CMakeFiles/onnxruntime_server.dir/build.make:216: src/CMakeFiles/onnxruntime_server.dir/onnx/value_info.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[1]: *** [CMakeFiles/Makefile2:117: src/CMakeFiles/onnxruntime_server.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2
```

**Comment by digoal** [comment]
```
root@73a7cc4cf7b3:/tmp/pg_onnx/onnxruntime-server/onnxruntime-server# gmake --version
GNU Make 4.3
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
root@73a7cc4cf7b3:/tmp/pg_onnx/onnxruntime-server/onnxruntime-server# cmake --version
cmake version 3.27.4

CMake suite maintained and supported by Kitware (kitware.com/cmake).
```

**Comment by kibae** [comment]
Hi. @digoal 🤗
Thank you for your interest in the project.

I'm surprised that this issue hasn't been detected in our test cases and CI phase. I'm working on fixing this issue, but [a recent flaw in onnxruntime 1.16.0](https://github.com/microsoft/onnxruntime/issues/17645) is preventing me from verifying it.
I believe you are installing a healthy onnxruntime. I've made a modification to the source code, would you like to try this, it's a temporary fix.
https://github.com/kibae/onnxruntime-server/pull/36/files

I'm on vacation, so my response may be late. But feel free to ask me any questions.
Thanks.

**Comment by kibae** [comment]
onnxruntime 1.15.1 and the above patch seem to be working fine.

#36 https://github.com/kibae/onnxruntime-server/actions/runs/6282216540/job/17061415241
Can you try it in your environment?

**Comment by digoal** [comment]
thank you , and here is another error when i use https://github.com/kibae/onnxruntime-server/pull/36

**Comment by kibae** [comment]
@digoal What Linux distributions are you running? [I've got Ubuntu and Debian covered](https://github.com/kibae/onnxruntime-server/blob/main/.github/workflows/cmake-linux.yml), but I'll have to run CI for the others.

**Comment by digoal** [comment]
debian 11.7
i changed the onnxruntime to 1.15.1 , it's OK now.
thank you.
best regards.

**Comment by kibae** [comment]
I'll try to catch up on 1.16.0 after my vacation.
Hopefully 1.16.1 will be released before then with the issues fixed. 😅

**Comment by lancerts** [comment]
The same issue happens for MAC ARM environment, v15.1 is fine.

**Comment by kibae** [comment]
@digoal @lancerts The issue was fixed in the onnxruntime-1.16.1 release, could you please check if you have the same problem?
#37
